compiler/compiler.py

```diff
@@ -427,3 +427,12 @@ def visit_Variable(self, node):
             code += "push dword [ebp]\n"
         self.stack_pos -= 4
         return code
+
+    def visit_ExpressionPopper(self , node):
+        old_stack_pos = self.stack_pos
+        code = ""+self.visit(node.expression)
+
+        code += "add esp, %i\n" % (old_stack_pos - self.stack_pos)
```

Comment:
check if `old_stack_pos - self.stack_pos` is zero before adding in the line of code.
This will prevent the appearance of the assembly line `add esp, 0`